setup.cfg

```diff
@@ -1,4 +1,5 @@
 [isort]
+default_section = FIRSTPARTY
```

Comment:
This is to ensure isort behaves like the old version (to minimize the diff, removing this changes a new 48 files).